res/layout/activity_note.xml

```diff
@@ -1,17 +1,54 @@
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
-    android:orientation="vertical" >
-
-    <EditText
-        android:id="@+id/text"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_margin="8dp"
-        android:inputType="textMultiLine"
-        android:minLines="2" >
-
-        <requestFocus />
-    </EditText>
-
-</LinearLayout>
\ No newline at end of file
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why did you add a scrollview here? Is scrolling the entire dialog ever necessary?